README.md

```diff
@@ -1,169 +1,156 @@
-$ variables by the relevant values).
+In addition to these external binaries, a native Z3 API for Linux is bundled
+with Inox and should work out of the box (although having an external Z3 binary
+is a good idea in any case). If you [build](#building-inox) yourself, the
+generated script should put the native API onto your classpath. Otherwise, you
+will have to make sure the relevant jar from [unmanaged](./unmanaged/) is on
+your runtime classpath.
+
+If a version for your platform is not shipped with Inox, you will have to
+recompile the native Z3 communication layer yourself; see the
+[ScalaZ3](https://github.com/epfl-lara/ScalaZ3) repository for information about
+how to build and package the project. You will then need to copy the resulting
+jar into the [unmanaged](./unmanaged/) directory, named
+`scalaz3-$os-$arch-$scalaBinaryVersion.jar` (replace the variables with the
+relevant values).
 
 ### Solver Defaults
 
-As of now the default solver is the native Z3 API. If that solver
-is unavailable, a series of fallbacks take place until the
-*princess* solver. You can specify which solver to use by
-e.g. giving the option
-``--solvers=smt-cvc4`` to use CVC4. Check the ``--solvers``
-line in Inox' help.
+Currently, the default solver is the native Z3 API. If that solver is
+unavailable, a series of fallbacks occur, ending with the *princess* solver. You
+can specify which solver to use by, for example, giving the option
+`--solvers=smt-cvc5` to use cvc5. Check the `--solvers` line in Inox's help for
+more details.
 
-Building Inox
--------------
+## Building Inox
 
-Inox is probably easiest to build on Linux-like platforms, but read on regarding other platforms.
+Inox is easiest to build on Linux-like platforms, but see below for other platforms.
 
-Due to its nature, this documentation section may not always
-be up to date; we welcome pull requests with carefully
-written and tested improvements to the information below.
+Due to the evolving nature of the project, this documentation may not always be up to date for all platforms; we welcome pull requests with carefully written and tested improvements to the information below.
 
 **Requirements:**
 
-* [Java SE Development Kit 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) or [Java SE Development Kit 7](http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html) for your platform
-* SBT 0.13.x (Available from http://www.scala-sbt.org/)
-* The [git](https://git-scm.com/) command on your path  
-  **Note:** this is *only* required for version computation and
-  you can avoid installing Git by using JGit (see the
-  [related documentation](https://github.com/sbt/sbt-git#using-jgit)
-  in `sbt-git`).
+* [Java Development Kit 17](https://openjdk.org/projects/jdk/17/) for your
+  platform (and the corresponding runtime environment)
+* sbt >= 1.6.0 (Available from http://www.scala-sbt.org/)
```

Comment:
Here you can add something like "SBT can also be installed by following the official Scala installation procedure (link)"